extensions/amp-list/amp-list.md

```diff
@@ -153,7 +153,7 @@ In `<amp-list>`, you can use the [`items`](#items) attribute to render a subset
 
 ### Specifying an overflow
 
-Optionally, the `<amp-list>` component can contain an element with the `overflow` attribute. Add an element with the AMP `overflow` attribute to `amp-list` if the AMP framework is unable to size it as requested. If you include a child element of `amp-list` with the `overflow` attribute, it will appear if one of the following conditions are met:
+Optionally, the `<amp-list>` component can contain an element with the `overflow` attribute. Add an element with the AMP `overflow` attribute to `amp-list` if the AMP framework is unable to size it as requested. If you include a child element of `amp-list` with the `overflow` attribute, it will appear if **none** of the following conditions are met:
```

Comment:
1. Don't think this statement is incorrect. In fact, line 158 should be `The bottom of amp-list is within the viewport`.
2. Can you also submit a fix in amp.dev repo to fix the external documentation?

Reply:
AFAIK this file will get imported into amp.dev.

Reply:
ah that's why I couldn't find it in the amp.dev repo. I started doubting my Code search skills ☹️

Reply:
🙂 The fact that the docs get imported is extremely far from obvious!

Reply:
If the bottom of amp-list is below viewport, it is free to resize. Therefore I think the addition of "none" is ok!

Reply:
What @samouri said is already mentioned on line 162.

Reply:
Not sure I completely follow. If you only modify line 158 to `The bottom of amp-list is within the viewport`, then you will have to also modify `one of` here to `all of`, and then change the second condition from `is within 15% ...` to `is not within 15%`, no? The overflow element should not display if the bottom of amp-list is within the viewport but *is* near the end of the page content (within 1000px or 15% of the height of the content). The code is here: 

https://github.com/ampproject/amphtml/blob/d1b027d1536f150e2eba714b527728b5f2bb844b/src/service/resources-impl.js#L991

Anyway, I think we should proceed with #32619 which IMO is better since it's in the positive tone.

Reply:
I agree. Let's proceed with #32619, I've already approved it. Thanks @samouri!